data/addressbook.json

```diff
@@ -6,70 +6,80 @@
     "studentId" : "00001",
     "tags" : [ "Class 4b" ],
     "parentPhoneNumberOne" : "91273527",
-    "parentPhoneNumberTwo" : "99149687"
+    "parentPhoneNumberTwo" : "99149687",
+    "classroom" "6 Innovation"
```

Comment:
Json storage format should either be class for form class, not "classroom"

Reply:
fixed